### PR TITLE
Support for Mirage F1

### LINKED ISSUE #2335: Mirage F-1
Before anyone else opens an issue for the newly released Mirage F-1, here's an overview on what needs to be done:

- [x] pydcs needs to be updated
- [x] creating yaml files in `resources/units/aircraft`
  * [x] Initially only the F1CE variant is available to fly
  * [x] BE, EE and M models will becomes available later, so preferably add support now...
  * [x] Making all AI assets available is quite a bit of work, for now I added the B, C-200, CT and EQ variants
- [x] create some squadrons for different variants
- [x] include variants of the aircraft in different factions (France, Spain, Iraq, Bluefor cold/modern)
- [x] add icons for air wing dialog
- [x] support for mission types (CAP/CAS/STRIKE/RUNWAY_ATTACK/OCA, except for C-200 being CAP only)
- [x] create banners
- [x] add payloads, special thanks to @Starfire13
- [x] radio support for Mirage-F1CE, special thanks to @jake-lewis 

As for the first point, I have a PR open at pydcs for the livery scanner which includes an export, so it shouldn't take too long until pydcs has updated either due to my PR or someone else's.

If I missed anything, feel free to point it out 👍

### PR DESCRIPTION
**Pull request by Raffson**
Starting as a draft for testers. Once complete it should resolve #2335.

### CODE REVIEW

**Review by DanAlbert — COMMENTED** [review]
pydcs update should be in its own PR

**Comment by DanAlbert** [comment]
And now I see this is a draft... No idea why GitHub bothers filling my inbox with drafts.

**Comment by Raffson** [comment]
> pydcs update should be in its own PR

That's a one-liner, why don't you take care of that ;)

**Comment by Raffson** [comment]
Special thanks @Starfire13 for supplying the loadouts!

**Comment by Raffson** [comment]
@DanAlbert I pretty confident the Mirage is ready, the last pydcs update makes sure it's flyable. I double checked and tested it live including the radios, for which we need to thank @jake-lewis.

File: game/radio/radios.py [hunk 1]
```diff
@@ -237,6 +237,15 @@ def __init__(self, frequency: RadioFrequency) -> None:
     Radio("AN/ARC-134", (RadioRange(MHz(116), MHz(150), kHz(25), Modulation.AM),)),
     # JAS39
     Radio("R&S Series 6000", (RadioRange(MHz(100), MHz(156), kHz(25), Modulation.AM),)),
+    # Mirage F1
+    Radio(
+        "V/UHF TRAP 136",
+        (
+            RadioRange(MHz(118), MHz(143), kHz(25), Modulation.AM),
```
Comment (DanAlbert) [inline game/radio/radios.py]:
```suggestion
            RadioRange(MHz(118), MHz(144), kHz(25), Modulation.AM),
```

File: game/radio/radios.py [hunk 2]
```diff
@@ -237,6 +237,15 @@ def __init__(self, frequency: RadioFrequency) -> None:
...
+            RadioRange(MHz(225), MHz(399), kHz(25), Modulation.AM),
```
Comment (DanAlbert) [inline game/radio/radios.py]:
```suggestion
            RadioRange(MHz(225), MHz(400), kHz(25), Modulation.AM),
```

File: game/radio/radios.py [hunk 3]
```diff
@@ -237,6 +237,15 @@ def __init__(self, frequency: RadioFrequency) -> None:
...
+        ),
+    ),
+    Radio("UHF TRAP 137B", (RadioRange(MHz(225), MHz(399), kHz(25), Modulation.AM),)),
```
Comment (DanAlbert) [inline game/radio/radios.py]:
```suggestion
    Radio("UHF TRAP 137B", (RadioRange(MHz(225), MHz(400), kHz(25), Modulation.AM),)),
```

**Pull request merged**